content/kit/pi/sd-card.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Flashing your SD card
+---
+
+To fully update your Pi's software, or refresh it if you think it's not working correctly, you can flash our SD card image onto the microSD card in your Pi. 
+
+To update the SD card, you'll need to download our update file, `pi-image.img.xz`. The flashing procedure is identical to flashing other images.
```

Comment:
Calling `pi-image.img.xz` an update file is confusing. Can we call it something else?